src/main/java/dev/langchain4j/automation/SayHelloCli.java

```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.automation;
+
+import com.github.rvesse.airline.annotations.Cli;
+import com.github.rvesse.airline.annotations.Command;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHUser;
+
+import java.io.IOException;
+
+@Cli(name = "@langchain4j-github-bot", commands = { SayHelloCli.SayHello.class })
+public class SayHelloCli {
+
+    interface Commands {
+
+        void run(GHEventPayload.IssueComment issueCommentPayload) throws IOException;
+    }
+
+    @Command(name = "sayHello")
+    static class SayHello implements Commands {
+
+        @Override
+        public void run(GHEventPayload.IssueComment issueCommentPayload) throws IOException {
+            // TODO check permissions
+            GHUser sender = issueCommentPayload.getSender();
```

Comment:
You can automatically check permissions with annotations: https://docs.quarkiverse.io/quarkus-github-app/dev/commands.html#permissions